app/config.sample/db.config.js

```diff
@@ -1,9 +1,9 @@
 const dbConfig = {
   USER: 'user-name',
-  PASSWARD: 'password',
```

Comment:
Nice 🏆

Reply:
You can use Spelling Checker extension for your IDE

Reply:
Ya give me link

Reply:
I use [this extension](https://github.com/streetsidesoftware/vscode-spell-checker)